### ISSUE TITLE
Cannot write \ in cells (text and code cells)

### ISSUE THREAD

**Issue by MoritzImendoerffer** [jupyter/notebook#2469]
I do not know why but suddenly the combination ALTGr + ? (equal to \) does not work anymore. The mouse cursor does change to a cross when pressing the combination but I cannot find a corresponding keyboard shortcut. As you can see from the title, I can write backslashes in the same browser.

Python 3.5.2 |Anaconda custom (64-bit)| (default, Jul  5 2016, 11:41:13) [MSC v.1900 64 bit (AMD64)]

Any ideas ?

**Comment by takluyver** [comment]
This may be related to #2379. There seems to be a regression for users of non-English keyboard layouts. Possibly to do with CodeMirror. Unfortunately, none of the core devs have the right kind of keyboard to reproduce the issue, so we need someone with such a keyboard to dive into it.

**Comment by jan-glx** [comment]
@takluyver No need to change the hardware keyboard, just change the layout through your os?

**Comment by Carreau** [comment]
No, we tried with changing layout. The browser uses location information of
keys not key code for the buggy part of the code. We actually need a non
English physical layout.

**Comment by jan-glx** [comment]
Oh ok. https://w3c.github.io/uievents/tools/key-event-viewer.html shows no difference between my Laptop's German QWERTZ and my USB US QWERTY for `minus` labeled `ß ? \` or `- _` nor for `RightAlt` labeled `AltGr` or `Alt` but between different Layouts and between chrome and FF...

**Comment by ruboerner** [comment]
I have found a workaround: Type "´" (acute) before AltGr-ß!

**Comment by MeTaNoV** [comment]
@ruboerner ThX for the tip but it would be better to fix this issue once and for all for the sake of the poor souls forced to use a German keyboard... (being German or Not)

**Comment by hroser** [comment]
Hi i am having the same problem and thank you for the workaround. Would be nice to have the issue fixed, on previous versions i did not have the problem.

**Comment by tschmidt83** [comment]
Same issue here, but the proposed Workaround is not working for me (MS Edge, Win10, using jupyter Notebook 5.0.0 out of anaconda 1.6.4).

**Comment by watercrossing** [comment]
Same issue here, but it works in Firefox and not in Chrome. UK keyboard with German keyboard layout, running UK Firefox on UK Win 8.1 works. Changing Windows keyboard layout to UK and it works in chrome too.

**Comment by takluyver** [comment]
If you're on notebook 5.0 or older, try upgrading to 5.1.

**Comment by watercrossing** [comment]
Yes, upgrading to `notebook-5.2.0` fixed the issue. Thanks.

**Comment by TranBao1992** [comment]
Just installed the notebook last week and I had this issue too. For me, the workaround is "^" though. Hope that helps. Still thanks @ruboerner for giving me the idea.

**Comment by jenssupan** [comment]
i take the asci code...
**_[alt]+92 --> \_**

**Comment by psinger** [comment]
Any update on this? Having same issue on newer jupyter versions.
